### ISSUE TITLE
[Settings - Startup]: Name is not defined for the button present under 'Startup' tab.

### ISSUE THREAD

**Issue by v-rpundir** [microsoft/terminal#13019]
### Windows Terminal version

1.13.10984.0

### Windows build number

22598.200

### Other Software

**Test Environment:**
**OS:** Windows 11 Version 21H2 (OS Build 22598.200)
**Screen Reader:** Narrator

### Steps to reproduce

**Repro Steps:**
Open Windows Terminal.
Open Settings page using "Ctrl + ',' ".
Verify that Name is not defined for the button present under 'Startup' tab.

**User Experience: -> Priority 1**
User who relies on screen reader will be affected and not be able to know what the control is and what is its functionality if name is not defined for the control.

**Guideline Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html

**Attachments:**
[Name is not defined for the button present under 'Startup' tab..zip](https://github.com/microsoft/terminal/files/8602787/Name.is.not.defined.for.the.button.present.under.Startup.tab.zip)

### Expected Behavior

Name should be defined for the button present under 'Startup' tab.
Screen reader should announce "When terminal starts button collapse/expand button".

### Actual Behavior

Name is not defined for the button present under 'Startup' tab.
Screen reader announces, "group collapse".

**Comment by zadjii-msft** [comment]
Hey look, @carlos-zamora was right, this one would get picked up.

Previous thread: #12317

**Comment by v-rpundir** [comment]
Closing as fixed now.
